<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Picture_view</class>
 <widget class="QDialog" name="Picture_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>33</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>899</width>
      <height>519</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>33</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>33</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #0F7DBE, stop:1 #1582C3);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>500</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>33</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>33</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu_Close">
    <property name="geometry">
     <rect>
      <x>868</x>
      <y>0</y>
      <width>32</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>关闭</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(238, 0, 0, 128), stop:1 rgba(238, 44, 44, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu_Min">
    <property name="geometry">
     <rect>
      <x>804</x>
      <y>0</y>
      <width>32</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>最小化</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(25, 134, 199, 0), stop:1 #50A3F0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu_Max">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>0</y>
      <width>32</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>33</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>最大化</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(25, 134, 199, 0), stop:1 #50A3F0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>33</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color:rgb(138, 138, 138);
background:transparent;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="prePicButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/icon/prev.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="largerButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/icon/larger.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="smallerButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/icon/smaller.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="nextPicButton">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/icon/next.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
